paddle/fluid/operators/bipartite_match_op.cc

```diff
@@ -147,6 +187,19 @@ class BipartiteMatchOpMaker : public framework::OpProtoAndCheckerMaker {
         "This tensor can contain LoD information to represent a batch of "
         "inputs. One instance of this batch can contain different numbers of "
         "entities.");
+    AddAttr<std::string>(
+        "match_type",
+        "(string, defalut: per_prediction) "
+        "The type of matching method, should be 'bipartite' or "
+        "'per_prediction', 'bipartite' by defalut.")
+        .SetDefault("bipartite")
+        .InEnum({"bipartite", "per_prediction"});
```

Comment:
it may be confusing that `match_type` is `InEnum({"bipartite", "per_prediction"})`.
Because, if `match_type ` is `per_prediction `, `BipartiteMatchOp` will use the two matching methods, but not only `per_prediction `.

Reply:
Yeah. Since the bipartite matching is always used. I do not change the operator name. And the `ArgMaxMatcher` here is based on the results of bipartite matching, I do not split it into two operators. Do you have any other suggestions?

Reply:
Do you think it is appropriate that replacing `match_type` with `follow_per_prediction`(bool)?